bundles/action/org.openhab.action.homematic/META-INF/MANIFEST.MF

```diff
@@ -10,7 +10,10 @@ Bundle-Activator: org.openhab.action.homematic.internal.HomematicActivator
 Bundle-ManifestVersion: 2
 Bundle-Description: This is the Homematic action of the open Home Aut
  omation Bus (openHAB)
-Import-Package: org.openhab.core.items,
+Import-Package: org.openhab.binding.homematic.internal.common,
+ org.openhab.binding.homematic.internal.communicator.client,
```

Comment:
@kaikreuzer is it true that an action bundle should not depend on a binding bundle even in the case where its actions functioning are intentionally dependent on a properly configured and running binding, where each action call's first parameter is an item bound to the binding?